icu4c/source/common/localebuilder.cpp

```diff
@@ -433,4 +433,9 @@ Locale LocaleBuilder::build(UErrorCode& errorCode)
     return product;
 }
 
+UBool LocaleBuilder::copyErrorTo(UErrorCode &outErrorCode) {
+    outErrorCode = status_;
+    return U_FAILURE(outErrorCode) ? TRUE : FALSE;
```

Comment:
U_FAILURE already returns a UBool.  No need to pass it through another ternary.

Reply:
DONE